emulate-browsers/base/injected-scripts/_descriptorBuilder.ts

```diff
@@ -0,0 +1,186 @@
+const nativeErrorRegex = new RegExp(/^(\w+):\s/);
+
+const globalSymbols = {};
+for (const symbol of ReflectCached.ownKeys(Symbol)) {
+  if (typeof Symbol[symbol] === 'symbol') {
+    globalSymbols[`${String(Symbol[symbol])}`] = Symbol[symbol];
+  }
```

Comment:
Why does this filename begin with an underline?

Reply:
I made the helper classes start with underlines just to indicate they weren't themselves "dom overrides"